packages/zipkin-transport-http/src/HttpLogger.js

```diff
@@ -11,11 +11,16 @@ const {
 } = require('zipkin');
 
 class HttpLogger {
-  constructor({endpoint, httpInterval = 1000, jsonEncoder = JSON_V1}) {
+  constructor({endpoint, headers = {}, httpInterval = 1000, jsonEncoder = JSON_V1}) {
     this.endpoint = endpoint;
     this.queue = [];
     this.jsonEncoder = jsonEncoder;
 
+    this.headers = Object.assign({
+      Accept: 'application/json',
```

Comment:
nit but we can save some overhead I think by nixing the Accept. it isn't usually required especially as we aren't parsing anything (and the server only returns json)

Reply:
:white_check_mark: